### PR TITLE
General improvements to the audit_math library

### PR DESCRIPTION
**Pull request by umbernhard**
**Description**

* Renames from `audits` to `audit_math
* Implementes typing for all of the audit_math files
* Cleans up some outdated documentation

**Testing**

N/A

**Progress**

Ready for review

### CODE REVIEW

File: audit_math/bravo.py [hunk 1]
```diff
@@ -65,7 +68,9 @@ def get_expected_sample_sizes(risk_limit, contest, sample_results):
         return math.ceil((weighted_alpha + (z_w / 2.0)) / (p_w * z_w + p_l * z_l))
 
 
-def get_test_statistics(margins, sample_results):
+def get_test_statistics(
+    margins: Dict[str, Dict], sample_results: Dict[str, int]
```
Comment (jonahkagan) [inline audit_math/bravo.py]: dunno if it's worth adding key/value types to the nested `Dict` in the type for `margins`

**Comment by jonahkagan** [comment]
Oh wait, it looks like `make typecheck` is failing on CI. I rescind my approval until that passes 😸

**Comment by umbernhard** [comment]
yep, helps to make sure the typing you implemented passes typecheck before you ask to merge :blush:

File: arlo_server/sample_sizes.py [hunk 2]
```diff
@@ -40,19 +40,6 @@ def get_sample_sizes(election: Election):
     sample_sizes = sample_size_options(election)
 
     # Convert the results into a slightly more regular format
```
Comment (jonahkagan) [inline arlo_server/sample_sizes.py]: prop don't need this comment anymore

**Pull request merged**